ehitchhiking-ui/src/app/shared/components/chat/chat.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="modal-wrapper">
+	<header class="modal-header">
+		<span class="modal-title">Chat</span>
+	</header>
+
+	<div class="chat" *ngIf="showChat">
+		<div class="message-zone">
+			<div *ngFor="let msg of msgList" class="message-wrapper" [class.my-message]="msg.isMy">
+				<div class="person-info">
+					<img [src]="msg.avaSrc" alt="photo" class="image" />
+					<div class="date">{{ msg.time | date: 'h:mm a' }}</div>
+				</div>
+				<div class="message">
+					<div class="text">{{ msg.text }}</div>
+				</div>
+			</div>
+		</div>
+
+		<form class="input-zone">
+			<mat-form-field class="input">
+				<input matInput #message [maxLength]="MAX_MESSAGE_LENGTH" placeholder="Message" id="messege-text" />
+				<mat-hint align="end">{{ message.value.length }} / {{ MAX_MESSAGE_LENGTH }}</mat-hint>
+			</mat-form-field>
+			<button class="send-btn" mat-raised-button (click)="onInput(message.value); onSend(message)">
```

Comment:
there should be only one function sendMessage

Reply:
done